gridData/__init__.py

```diff
@@ -3,8 +3,7 @@
 # Released under the GNU Lesser General Public License, version 3 or later.
 # See the files COPYING and COPYING.LESSER for details.
 
-"""
-:mod:`gridData` -- Handling grids of data
+""":mod:`gridData` -- Handling grids of data
```

Comment:
Flair wise I personally feel like it looks better to invert this i.e.
```suggestion
"""Handling grids of data -- :mod:`gridData`
```

Reply:
changed